packages/desktop-client/src/components/settings/Export.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { send } from 'loot-core/src/platform/client/fetch';
+import { Text, Button } from 'loot-design/src/components/common';
+
+import { ButtonSetting } from './UI';
+
+export default function Export({ prefs }) {
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏could we rename this component to something else? I find keywords such as `export`, `import`, `var`, `let`, `const`, etc. to be reserved irrelevant the casing.

Perhaps `ExportSection`?